### PR TITLE
fix: the post-fan-out batch — 23 cross-lane items, plus #47 and #50

### LINKED ISSUE #47: Lift Biome's packages/ and scripts/ exclusions and run the one-shot format
Follow-up scheduled by KC-PR-#39 and re-flagged by KC-PR-#45: `biome.json` excludes `packages/` and `scripts/` from `files.includes`, so `make lint` and the pre-commit hook silently skip the whole packages tree. This was deliberate lane-scoping while parallel children were writing packages/ (each lane self-formatted to the repo's Biome settings), not an oversight — but it must not outlive the parallel phase.

**When:** after #42 (which also touches `biome.json`) and the in-flight core lanes (#11) merge — a two-line diff plus one `make format` commit, done in a quiet window with zero children in flight so the format commit can't conflict with anyone.

**Acceptance criteria**
1. `biome.json` carries no packages/scripts exclusion; `make lint` demonstrably checks a file under `packages/core/src` (prove with a temporary violation).
2. The one-shot format commit is separate from the config change, mechanical, and touches no logic.
3. The vendored `apps/web/src/components/ui/**` exclusion stays until #6's kit replaces those files (deleted by later UI children).

Part of #1's toolchain hygiene, not an epic child.

### LINKED ISSUE #50: packages/core and packages/app have no coverage provider — the 80% floor can't be measured
Surfaced by KC-PR-#49 (and worked around via temporary symlinks in KC-PR-#44/#45/#48): `@vitest/coverage-v8` is a devDependency of `apps/web` only, so `vitest run --coverage` cannot run in `packages/core` or `packages/app` — the session-completion checklist's ≥80% touched-file floor is argued, not measured, on every core/app PR.

**Fix:** add `@vitest/coverage-v8` (workspace-consistent version) to both packages' devDependencies + a `test:coverage` script; wire an optional coverage step in `pr.yml` if cheap. One lockfile change — schedule with #47's quiet-window chore so it can't conflict with in-flight lanes.

**Acceptance criteria**
1. `pnpm --filter @kro/core test:coverage` and `--filter @kro/app test:coverage` run without symlink hacks.
2. The next feature PR states a measured touched-file percentage.

### LINKED ISSUE #71: Post-fan-out batch: cross-lane one-liners parked by the feature-UI wave
Parked cross-lane needs reported by the fan-out builders (lanes were busy; per the Kotoamatsukami bar these route to a batch instead of parallel edits). Work in a quiet window after the wave merges:

From KC-PR-#67 (Do UI):
1. `design/system/icons/icons.ts`: widen `LiquidGlassFAB*` glyph typing to `KitSymbolName` (needs `bolt.fill`, `clock.badge.xmark`) — then switch the Do FAB from the `target` stand-in to canon's `bolt.fill`.
2. `features/do/DoFeature.ts`: widen `userDidTapCard`'s lane to canon's signature (events-allday/events-timed excluded today; removes a cast in DoPage).
3. `features/do/DoProducer.ts`: fold DoOverflowProducer's five thunks in with reducer arms so failed overflow writes reach the banner.
4. `MainShellFragment.tsx`: desktop toolbar order to canon's Inbox, Refresh, Visibility; consider the full-bleed gradient variant that drops #67's three adaptations.
5. `check-uzf-boundaries.mjs`: a remaining RC-50 string where the rule is RC-40.
6. Strip the now-superseded inline glass.css workarounds from the capture (#64) and do (#67) surfaces once the kit hotfix PR is merged.

Related standing items: #47 (Biome exclusions lift + one-shot format), #50 (coverage provider for packages). Batch them into the same quiet window.

**Acceptance criteria.** Each line lands as one small commit in one PR; the Do FAB draws bolt.fill; no inline glass workarounds remain; the wave's PR-body notes referencing these items are satisfied.

### PR DESCRIPTION
**Pull request by zheref**
> 🟥 **Ichigo · Substitute** — the local plane, entire · *local, on your creds · I open PRs for **you** to merge (I never merge `main`, never review my own work)* — acting as **Shinigami** (product code).

Closes #71
Closes #47
Closes #50

# What this changes for you

**Undo actually appears.** Every merged surface already enqueued toasts — the capture Undo, the completion Undo, the triage banner — and none of them were drawn, because the one host that renders them lived inside two feature surfaces that the shell does not mount. It is now mounted once, at the shell, above every destination. Capturing the proof found a second defect nobody had seen: on the desktop (sidebar) shell the toast was anchored to the *viewport*, so it began underneath the sidebar with its message clipped. It now starts at the content column.

**The Do FAB is canon's bolt.** `target` was a stand-in; Clear Expired was a stand-in too (`clock.badge.xmark` now). The desktop toolbar runs Inbox → Refresh → Visibility, the order canon fixes.

**Lists destinations list something.** `FindProducer` never hydrated a row's list from the project store, so *every* row was filtered out of every list destination — the destinations existed and were permanently empty. Long-press now offers **View Detail**, and **Share** is a real `ShareService` behind `ThunkExtra` (Web Share API, clipboard fallback) rather than a named refusal.

**Plan's moves survive.** A quadrant assignment is written to disk instead of living in memory until the next pool read; Start Session carries the row's identity into Execute instead of opening an anonymous session; the capability flags come from a real resolver Producer instead of a hard-coded `true`; and the list's sort/grouping now read the user's Settings choice — which the slice's mirror had been silently ignoring.

**Signed in stays signed in.** `observeAuthState` was written and never wired: a token refresh, a sign-out in another tab, or an expiry never reached the store. The composition root observes it now.

**And the repo lints itself again.** `packages/**` and `scripts/**` had been excluded from Biome; lifting the exclusion surfaced 88 real findings (49 fixed, 3 rule decisions recorded in `TOOLCHAIN.md`, 15 interactive-a11y ones tracked in KC-IS-#77). Both packages have a coverage provider and a `make test-coverage` verb.

---

## Disposition — every item in KC-IS-#71, plus #47 and #50

| # | Item | Disposition |
|---|---|---|
| 15 | Mount `ActiveToastHost` at the shell | **Done** — mounted once in `MainShellPage`, above `ToolbarSlotsProvider`; the two feature-level hosts deleted. Proven by capture. |
| 1 | Widen FAB glyph typing to `KitSymbolName`; Do FAB → `bolt.fill` | **Done, resolved the other way.** The FAB is `bolt.fill` and Clear Expired is `clock.badge.xmark`. The *widening* is **skipped as wrong**: promoting those four rows into the shared table (item 8/13/16's work) makes `SfSymbolName` sufficient. Widening instead would have inverted the chrome→endeavor layering to buy nothing. |
| 2 | `userDidTapCard` carries canon's lane | **Done** — payload is `{ section, endeavorId }`; the cast in DoPage is gone. |
| 3 | Fold `DoOverflowProducer` into `DoProducer` with reducer arms | **Done** — five thunks folded, `OVERFLOW_THUNKS` + two `addMatcher` arms write `withException`, so a failed overflow write reaches the banner. A vacuous test surfaced doing it (a Delegate case that asserted against a never-loaded day) and was rewritten to canon's actual rule. |
| 4a | Desktop toolbar order | **Done** — Inbox, Refresh, Visibility. Captured. |
| 4b | Full-bleed gradient variant dropping #67's three adaptations | **Skipped — a human visual call.** Those three adaptations are *measured* contrast adaptations against `contrastContracts.ts`; dropping them without a designer's eye trades a proven contract for an unproven look. Left as-is rather than guessed. |
| 5 | `check-uzf-boundaries.mjs` says RC-50 where the rule is RC-40 | **Done** — message and header comment both say `RC-40`; `spec/architecture/web.md` carries the citation note. |
| 6 | Strip the inline glass workarounds (#64, #67, #70) | **Done** — all inline `position`/`isolation` workarounds removed now that `styles.css` imports the layer. |
| 7 | Wire `observeAuthState` in `providers.tsx` | **Done** — store + extra built in one `useState`, observer started in a `useEffect` and torn down with it. |
| 8 | Promote the ~30-row settings glyph map | **Done** — folded into `icons.ts`; `settingsIcons.ts` is a resolver. |
| 9 | Strip #70's three inline glass workarounds | **Done** (with item 6). |
| 10 | `docs/Features/Settings.md` | **Done** — language-agnostic, with mermaid. |
| 11 | **Lists destinations list nothing** | **Done** — `readStoredEndeavors` reads `localStore.projects.all()`, builds `projectsById`, and passes `list` into `endeavorFromRecord`; `EndeavorRelations` gained `list`. |
| 12 | `contextMenu` `viewDetail` binding | **Done** — in `@kro/core`'s operations, surfaced by Find's long-press. |
| 13 | Nine SF-Symbol rows + the PropertyRow colour swatch | **Done** — rows in `icons.ts`; the swatch role in `contrastContracts.ts`. |
| 14 | `docs/Features/Find.md` | **Done.** |
| 16 | Eight SF Symbols + three `contrastContracts` rows | **Done** — `LABEL_ON_FILL_ROLES` (completeBlue / focusGreen / pastryGreen, black label) is now measured and under contract. |
| 17 | A **browser** regression test for the glass layering | **Done** — `apps/web/e2e-kit/glass-layering.spec.ts`, four tests, asserting the `::before` backdrop filter and `isolation: isolate` in a real browser (jsdom computes `backdropFilter: none` and would have passed vacuously). |
| 18 | `ShareService` behind `ThunkExtra` | **Done** — interface + live/stubbed + fixtures, one line on the manifest. `ShareOutcome` lives in `@kro/core` so no Producer imports `services/**` (the boundary check refuses it). |
| 19 | `PlanState` gains `listSort`/`listGrouping` | **Done, resolved the other way — the mirror is deleted, not grown.** The slice's copy had no sync path: it was seeded once and then silently ignored the user's Settings choice for the rest of the session. Four `createSelector` preference selectors compose over `selectSettingValues` instead, so the list reads the one source. |
| 20 | Persist a quadrant assignment | **Done** — `persistQuadrantAssignmentsThunk` writes the row the *slice* resolved (never a second derivation), skips a vanished id rather than failing the batch, and resolves an error rather than throwing. Now covered end to end from the Page. |
| 21 | Start Session carries the endeavor | **Done** — `prepareSessionLaunchThunk` runs *before* the navigation, so Execute never paints an anonymous frame first. |
| 22 | A real capability-flag resolver Producer | **Done** — `resolvePlanFlagsThunk` returns `{ isQuickEventCreationEnabled, enabledCapabilityFlags }`; the Page dispatches the plain event with the answer, so no Selector reaches a flag Service. |
| 23 | Relocate `PlanListSelectors`/`Producer` into the feature's suffixed files | **Done** — `pages/list/` no longer holds a Selector or a Producer. |
| **#47** | Lift Biome's `packages/`+`scripts/` exclusions, one-shot `make format` | **Done, and larger than its estimate.** 88 findings, not "a two-line diff": 49 fixed, 3 rule decisions recorded with reasons in `TOOLCHAIN.md`, 15 interactive-a11y findings tracked in KC-IS-#77 (downgraded to `warn` for `packages/app/src/**`, not silenced). The vendored `components/ui` exclusion **stays, because the vendored files do**: `apps/web/src/components/ui/` still holds 56 Chakra-snippet files, imported by the `(legacy)` route group that #22 retires. Linting generated third-party snippets would produce findings nobody may act on. It is the only surviving exclusion, and it is scoped to that one directory. The format pass is its own mechanical commit (`f679ebf`). |
| **#50** | Coverage provider for both packages | **Done** — `@vitest/coverage-v8` pinned to the Vitest minor, `test:coverage` on both packages, `make test-coverage`, numbers below. |

### The three rule decisions #47 forced

Recorded in `TOOLCHAIN.md` under *"Rules that are not at their preset severity, and why"* — not silently disabled:

- `a11y/useSemanticElements` → **off** (repo-wide). It demands `<output>` for `role="status"`; the kit's live regions are `<div role="status">` by design because `<output>` carries form-association semantics the toast has no business claiming.
- `a11y/noStaticElementInteractions` → **warn** for `packages/app/src/**` (15 findings, KC-IS-#77).
- `a11y/useKeyWithClickEvents` → **warn**, same scope, same issue.

---

## Coverage — measured, both packages

`make test-coverage` (v8 provider). Whole-package:

| Package | Lines | Branches | Functions |
|---|---|---|---|
| `@kro/core` | **97.56 %** | 94.56 % | 95.59 % |
| `@kro/app` | **95.65 %** | 91.73 % | 90.56 % |

Touched files, line coverage:

| File | Lines |
|---|---|
| `design/system/icons/icons.ts` | 100 % |
| `features/settings/pages/settingsIcons.ts` | 100 % |
| `features/capture/pages/captureIcons.ts` | 100 % |
| `features/triage/pages/triageIcons.ts` | 100 % |
| `design/endeavor/endeavorIcons.ts` | 100 % |
| `design/endeavor/CardBadge.tsx` | 100 % |
| `design/endeavor/PropertyRow.tsx` | 100 % |
| `features/plan/PlanSelectors.ts` | 100 % |
| `features/plan/PlanState.ts` | 100 % |
| `features/find/FindState.ts` · `FindShifters.ts` | 100 % |
| `features/find/FindProducer.ts` | 98.03 % |
| `services/platform/share/ShareService.ts` | 97.59 % |
| `features/do/DoProducer.ts` | 97.51 % |
| `features/do/DoFeature.ts` | 95.32 % |
| `features/find/FindFeature.ts` | 95.05 % |
| `features/plan/PlanProducer.ts` | 92.91 % |
| `features/plan/pages/PlanPage.tsx` | **80.37 %** |

**One exemption, named.** `PlanPage.tsx`'s residue (lines 844–862) is the **Radix popover** frame of the add-existing picker — the desktop presentation. This repo cannot mount a Radix popper under jsdom: `packages/app/src/design/system/primitives/__tests__/radixEnvironment.tsx` documents the measurement (5–12 s per mount, growing per mount, `[vitest-worker]: Timeout calling "onTaskUpdate"`) and the rule it forces — popper panels are covered by a real browser, never by `make test`. The sheet branch, the confirm-and-write path and the Escape dismissal are all covered; the file went from **72.02 %** to **80.37 %** in this PR.

---

## How to verify

Each block stands alone. `make setup` first if this is a fresh clone.

### Item 15 — the toast is mounted, and it clears the chrome

```
pnpm --filter @kro/web dev
```

1. Open `/my-day`. Add a task through the FAB → Quick Add.
2. Complete it (prepare the card, **Mark complete**, then **Mark**). The Undo toast appears **over My Day** — it did not exist before this PR.
3. Start a session (**Start Session** on a row). The session pill appears; complete another row. The toast now sits **above the pill**, not under it.
4. Widen the window past 1024 px so the sidebar shows. The toast starts at the **content column**, not under the sidebar — that was the second defect the capture found.

Automated: `pnpm --filter @kro/app exec vitest run src/features/main/__tests__/MainShellPage.test.tsx src/design/chrome/toast`.

### Item 1, 4a — the FAB and the toolbar

Same dev server. The Do FAB draws a **bolt**, not a target; its menu's Clear Expired draws a **clock with a badge**. On desktop, the toolbar reads **Inbox, Refresh, Visibility** left to right.

### Items 8, 13, 16 — one symbol table

```
pnpm --filter @kro/app exec vitest run src/design/system/icons src/features/settings/pages/__tests__/settingsIcons.test.ts src/features/capture/pages/__tests__/captureIcons.test.ts src/features/triage/pages/__tests__/triageIcons.test.ts src/design/system/tokens
```

The disjointness tests prove no lane map re-declares a row the shared table owns; `contrastContracts` proves the three new label-on-fill roles measure over 4.5:1.

### Item 11 — lists list something

```
pnpm --filter @kro/app exec vitest run src/features/find
```

Or in the browser: create a project, put a task in it, open the list destination. Before this PR it was empty for every project.

### Items 12, 18 — long-press Detail, and a real Share

```
pnpm --filter @kro/app exec vitest run src/features/find src/services/platform/share
```

In the browser, long-press a row in Find: **View Detail** is offered, and **Share** either opens the OS share sheet or copies the row and says so.

### Items 19–23 — Plan

```
pnpm --filter @kro/app exec vitest run src/features/plan
```

In the browser: switch Plan to **Priority Matrix**, use a quadrant's **Add existing**, pick a task, confirm — then reload. The task is still in that quadrant (it was not, before). On a row's overflow, **Start Session** opens Execute already showing that row's title and glyph. Change *List sort* in Settings → Plan's list follows immediately.

### Item 7 — the auth observer

```
pnpm --filter @kro/web exec vitest run src/app/__tests__
```

In the browser: sign in, then sign out in a second tab. The first tab follows.

### Item 17 — the glass layering, in a real browser

```
pnpm --filter @kro/web exec playwright test --project=kit e2e-kit/glass-layering.spec.ts
```

Four tests. They read `getComputedStyle(node, '::before').backdropFilter` — jsdom reports `none` there and would pass vacuously, which is exactly why this one is a browser test.

### #47 — lint really does bite `packages/`

Proven by breaking it on purpose (and reverting):

```
printf 'const x: any = 1\nexport default x\n' > packages/core/src/__lintprobe.ts
make lint            # fails: lint/suspicious/noExplicitAny in packages/core
rm packages/core/src/__lintprobe.ts
make lint            # 0 errors
```

### #50 — coverage

```
make test-coverage
```

`@kro/core` 97.56 % lines, `@kro/app` 95.65 % lines.

### Everything

```
make lint && make typecheck && make test && make build
pnpm --filter @kro/web run build-storybook
```

All green locally: **9 305** tests (core 1 851 · web 135 · app 7 319), lint 0 errors, build and Storybook clean.

---

## Screenshots

Real renders of the built app (`next start`, Playwright), both viewports and both themes. Mock-fixture data only.

### The Active Toast host at the shell (item 15)

| | Over My Day | Lifted above the session pill |
|---|---|---|
| **Mobile 390×844 · light** | toast over My Day, mobile light | toast lifted above the pill, mobile light |
| **Mobile 390×844 · dark** | toast over My Day, mobile dark | toast lifted above the pill, mobile dark |
| **Desktop 1440×900 · light** | toast over My Day, desktop light | toast lifted above the pill, desktop light |
| **Desktop 1440×900 · dark** | toast over My Day, desktop dark | toast lifted above the pill, desktop dark |

The desktop pair is also the evidence for the defect the capture found: the toast starts at the content column, not at the window edge under the sidebar.

### The Do FAB — `bolt.fill`, and `clock.badge.xmark` in its menu (item 1)

| | Closed | Open |
|---|---|---|
| **Mobile 390×844 · light** | Do FAB closed, mobile light | Do FAB open, mobile light |
| **Mobile 390×844 · dark** | Do FAB closed, mobile dark | Do FAB open, mobile dark |
| **Desktop 1440×900 · light** | Do FAB closed, desktop light | Do FAB open, desktop light |
| **Desktop 1440×900 · dark** | Do FAB closed, desktop dark | Do FAB open, desktop dark |

### The desktop toolbar — Inbox, Refresh, Visibility (item 4a)

| | Canon's order |
|---|---|
| **Desktop 1440×900 · light** | desktop toolbar in canon's order |

The capture spec that produces all seventeen is committed at `apps/web/e2e/kc71-shots.spec.ts` (skipped unless `KC71_SHOTS` is set, so it never runs in CI):

```
pnpm --filter @kro/web build && pnpm --filter @kro/web exec next start -p 3111 &
KC71_SHOTS=1 PLAYWRIGHT_PORT=3111 pnpm --filter @kro/web exec playwright test e2e/kc71-shots.spec.ts
```

---

## Definition of Done

- [x] `make lint && make typecheck && make test && make build` green locally; `build-storybook` green.
- [x] Per-artifact minimums met for every new Selector, Shifter, Producer and Service; new behaviour covers the happy path and ≥1 edge case.
- [x] ≥80 % line coverage on every touched file — **one exemption named above** (`PlanPage.tsx`'s Radix popover frame, with the repo's own measurement of why).
- [x] Tests use `stubbedThunkExtra` / `stubbed…Service`; no live network, no mocked `fetch`, no inline `State`.
- [x] No new `any`, no `@ts-ignore`; every `biome-ignore` states its reason on the same line.
- [x] Conventional Commits, one focused commit per item; nothing committed with `--no-verify`.
- [x] `docs/Features/Settings.md` and `docs/Features/Find.md` authored with mermaid diagrams (items 10, 14).
- [ ] **Feature flags** — exempt: every item here is a fix, a wiring completion or a refactor of already-shipped, already-flagged behaviour. No new user-visible surface was introduced.

Follow-up filed, not carried: **KC-IS-#77** (15 interactive-a11y findings, downgraded to `warn` rather than silenced).

Bankai-Agent: ichigo

### CODE REVIEW

File: packages/core/src/utils/numberEnhancements.ts [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 declare module 'numberEnhancements' {
-    interface Number {
-        get seconds(): number;
-        get minutes(): number;
-        get hours(): number;
-    }
+  interface Number {
+    get seconds(): number
+    get minutes(): number
+    get hours(): number
```
[the remark this answers is not in the document]
Comment (zheref) [reply packages/core/src/utils/numberEnhancements.ts]: Agreed, and deleted in 23dbc4c — with its side-effect import in `SessionConfig.ts`.

Worth naming precisely, because it was worse than dead: `declare module 'numberEnhancements'` augments a *phantom module of that name*, not the global `Number`, so even with the implementation uncommented `1.minutes` would not have compiled. The real helpers are `utils/durations` (`seconds`/`minutes`/`hours`/`secondsFromMinutes`), which `SessionConfig` already imports and uses — `preset.minutes` there is a plain field, not an extension.

`@kro/core`: 114 files, 1851 tests, still green.

**Comment by zheref** [comment]
## Standing in for the second automated reviewer — Bugbot is quota-capped

`Cursor Bugbot` reported **"Bugbot couldn't run - usage limit reached"** on this PR, so its round does not exist. Rather than let a 690-file diff go out with one reviewer, here is the pass it would have made, with what it found. Findings are stated as findings; the two real ones are already fixed on this branch.

### 🔴 Found and fixed — an aborted flag resolve was not silent (`e51c864`)

Item 22's mount effect in `PlanPage` aborts on unmount, and then **stamped the day anyway**:

```ts
const effect = dispatch(resolvePlanFlagsThunk())
void effect.then((action) => {
  const result = resolvePlanFlagsThunk.fulfilled.match(action) ? action.payload : null
  dispatch(onViewLoaded({ …, isQuickEventCreationEnabled: result?.ok === true ? … : false }))
})
return () => effect.abort()
```

An aborted thunk resolves `rejected`, `result` is `null`, and the cancelled mount writes `onViewLoaded` with **the flags off** — racing the live mount's own answer for the last write. React's development double-invoke makes that an everyday path, not a corner. Canon is explicit that cancellation is the one silent exit (`UZF-14`; this repo's mirror: *"the `.rejected` arm returns early on `action.meta.aborted`"*). Guarded now, with a regression test that was **verified to fail without the guard** (`expected 1788278544371 to be +0`) rather than assumed to.

### 🔴 Found and fixed — a dead ambient module (`23dbc4c`, Copilot's find, extended)

`numberEnhancements.ts` was worse than dead: `declare module 'numberEnhancements'` augments a *phantom module of that name*, not the global `Number`, so `1.minutes` would not have compiled even with the commented-out implementation restored. Deleted with its side-effect import.

### 🟢 Checked, no finding

- **`ShareService`** — `navigator.share` is **bound** rather than referenced (a detached call throws `Illegal invocation`, which is the classic way an availability check passes and the share still fails). Cancel (`AbortError`) does **not** fall through to the clipboard: copying something the user just declined is a second action they did not ask for. A *failed* share does, because a failed hand-off with an empty clipboard leaves them nothing. Non-secure origins have neither API and land on `unavailable`. Nothing is written anywhere the user did not aim it.
- **`observeAuthState` at the composition root** — subscribed in a `useEffect` whose cleanup **is** the teardown, and handed the **same** `ThunkExtra` the store's Producers read. A second `liveThunkExtra` spread would have subscribed to a different Supabase client than every thunk uses; that is why the extra is kept beside the store rather than rebuilt.
- **`check-platform-free.mjs`** — the `while ((m = re.exec(…)))` → `matchAll` change preserves semantics (both regexes are global; `matchAll` clones, so no `lastIndex` leak) and the ban list is untouched. The guard is not weakened by the lint sweep.
- **`persistQuadrantAssignmentsThunk`'s `getState()`** — a narrow, named read (`UZF-15`), deliberate: the row written is the one the **slice** resolved, so what is on screen and what is on disk cannot come from two derivations. A vanished id is skipped rather than failing the batch; a write failure resolves `err(...)` rather than throwing.
- **Biome a11y downgrades** — `noStaticElementInteractions` and `useKeyWithClickEvents` are `warn`, not `off`, scoped to `packages/app/src/**`, with the 15 findings tracked in KC-IS-#77. `useSemanticElements` is off repo-wide with its reason in `TOOLCHAIN.md` (it demands `<output>` for `role="status"`; a toast has no form association to claim).
- **No secrets, no PII to a third party, no new network surface.** The one new Service is a browser capability; the fixtures are mock data.

### ⚠️ Named, not fixed — the one thing a reviewer should still weigh

The diff is **690 files**, because #47's exclusion lift means one mechanical `make format` touches the whole newly-included tree. That commit (`f679ebf`) is **pure formatting and nothing else** — it was ordered after the config and the fixes precisely so it could be read as a no-op. Reviewing it hunk by hunk is not a good use of a human's attention; reviewing the other 19 commits is.

**Comment by zheref** [comment]
**Round 2 (Copilot) — addressed.** The finding is right, and it was a defect in the PR body rather than in the config.

`biome.json` does still disable the linter for `apps/web/src/components/ui/**`, and it should: **56 Chakra-snippet files are still there**, still imported by the `(legacy)` route group (`app/(legacy)/layout.tsx` pulls `app-providers`, `color-mode`, `navigation-layout` and `toaster` from it). `CLAUDE.md` names that set as vendored and #6/#22's to retire. Linting generated third-party snippets produces findings nobody may act on, so the exclusion stays until the files go.

What was wrong was my sentence claiming the exclusion was gone. **The body is corrected** — it now says the exclusion stays, why, and that it is the only surviving one, scoped to that one directory.

The other two exclusions #47 named — `!packages` and `!scripts` — are genuinely lifted, and `make lint` was proved to bite `packages/` by planting a violation and watching it fail (the recipe is in **How to verify → #47**).

**Comment by zheref** [comment]
**Kotoamatsukami bar audit passed** — MERGEABLE/CLEAN, CI green, 9306 tests, Copilot's two rounds addressed+resolved (incl. its catch of a false claim in the PR's own body — corrected), Bugbot quota-capped with a disciplined self-review in its place (whose second defect — the aborted flag-resolve racing the live mount — was fixed with a proven-failing regression), 17 screenshots published and rendering, measured coverage quoted for both packages, gate verdict quoted with the structural residue. Two decisions recorded for the maintainer's read, not questions: item 4b (full-bleed gradient) deliberately left as a designer's call, item 19 resolved by deleting the sync-less mirror. The worker filed KC-IS-#77 for the 15 residual Biome findings.

**Held at G2 for the maintainer**: closes #71 + #47 + #50 in one merge.
